core/src/main/kotlin/core/io/VsqLike.kt

```diff
@@ -36,7 +36,7 @@ object VsqLike {
 
     suspend fun match(file: File): Boolean {
         val midi = Mid.parseMidi(file)
-        val midiTracks = midi.track as Array<dynamic>
+        val midiTracks = midi.tracks as Array<dynamic>
```

Comment:
Is it happening after we replaced midi library?

Reply:
Yes. I think it's better to write typing of `midi-file`

Reply:
If you have time ^^